test/shared/PosmTestSetup.sol

```diff
@@ -75,4 +82,13 @@ contract PosmTestSetup is Test, Deployers, DeployPermit2, LiquidityOperations {
             delta = delta + hook.deltas(i);
         }
     }
+
+    function _getPositionLiquidity(uint256 tokenId, PositionConfig memory _config)
```

Comment:
this PR introduces a library that does exactly this, feels wrong that our tests that integrate then cant access it and youve had to copy the same code into the tests?
maybe its worth exposing a public function on position manager called `getPositionLiquidity(tokenId, config)` that returns the liquidity? and then tests/integrations can use that?
thoughts?

Reply:
mmm im not sure i like that given its to be used internally? like we dont use the StateLibrary internally in PM? but i dont feel strongly